lnrpc/lightning.proto

```diff
@@ -2189,6 +2189,33 @@ message OpenChannelRequest {
     attempted.
     */
     bool scid_alias = 20;
+
+    /*
+    The base fee charged regardless of the number of milli-satoshis sent.
+    */
+    uint64 base_fee = 21;
+
+    /*
+    The effective fee rate in milli-satoshis. The precision of this value
+    goes up to 6 decimal places, so 1e-6.
+    */
+    uint64 fee_rate = 22;
```

Comment:
Not sure if what we want here is the amount in milli-satoshis or the same granularity that we have in the config file (ppm directly)

https://github.com/lightningnetwork/lnd/blob/master/sample-lnd.conf#L524